internal/testrunner/coverageoutput.go

```diff
@@ -113,6 +125,101 @@ func (c *coberturaCoverage) bytes() ([]byte, error) {
 	return buffer.Bytes(), nil
 }
 
+// Merge merges two coverage reports for a given class.
+func (c *CoberturaClass) Merge(b *CoberturaClass) {
```

Comment:
Do you see feasible adding unit tests for this method? I see a lot of calculations that would benefit for the tests

Reply:
Done